packages/kbn-i18n/src/angular/directive.ts

```diff
@@ -36,9 +40,10 @@ export function i18nDirective(i18n: I18nServiceType): IDirective {
           $element.html(
             i18n(id, {
               values,
-              defaultMessage,
+              defaultMessage: $sanitize(defaultMessage),
             })
           );
+          $compile($element.contents() as any)($scope.$parent);
```

Comment:
@azasypkin I believe now it is safe to compile html because:
1) we have agreed not to use tags in `defaultMessage`.
2) even if some vendor pass any tags, scripts or directive in `defaultMessage` for some language, before inserting `defaultMessage` to DOM we check passed string using `$sanitize` angular service which removes all unsafe tag like `script` and directives.
For example we have such text in `defaultMessage`:
```javascript
<span>hello</span>world
<im-dangerous-directive>I will destroy something</im-dangerous-directive>
<script>document.write('ha-ha')</script>
```
after using $sanitize service we will get:
```javascript
<span>hello</span>world
```